devices/debian.py

```diff
@@ -966,6 +966,11 @@ def link_up(self, interface):
         else:
             return None
 
+    def set_link_status(self, interface, status):
```

Comment:
Can we rename this to set_link_state? This is just turning the interface on and off, right?

Reply:
Sure will rename the same